docs/reference/oras-auth-provider.md

```diff
@@ -143,69 +183,64 @@ EOF
-
-#### User steps to set up Managed Identity with AKS and ACR:
-
-1. Create ACR
-2. Create AKS cluster
-3. Configure managed identity on the AKS managed VMSS.  
-   a. Directly using kubelet identity:
-      If your AKS cluster has already run `az aks update --attach-acr` to ensure access to the private ACR, kubelet identity will directly has the permission to pull artifacts.
-      ```shell
-      export IDENTITY_CLIENT_ID=$(az aks show -g ${AKS_CLUSTER_NAME} -n ${RESOURCE_GROUP} --query "identityProfile.kubeletidentity.clientId")
-      ```
-   b. Bring your own user-assigned managed identity:
-      ```shell
-      # Create one identity
-      az identity create --name ${IDENTITY_NAME} --resource-group ${RESOURCE_GROUP}
-      export IDENTITY_CLIENT_ID=$(az identity show --name myIdentity --resource-group ${RESOURCE_GROUP} --query 'clientId' -o tsv)
-      export IDENTITY_PRINCIPAL_ID=$(az identity show --name myIdentity --resource-group ${RESOURCE_GROUP} --query 'principalId' -o tsv)
-      export IDENTITY_RESOURCE_ID=$(az identity show --name myIdentity --resource-group ${RESOURCE_GROUP} --query 'id' -o tsv)
-
-      # Assign identity ACR pull permission to ACR
-      az role assignment create --assignee ${IDENTITY_PRINCIPAL_ID} --role "Acrpull" --scope "/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP}/providers/Microsoft.ContainerRegistry/registries/${ACR_NAME}"
-
-      # Configure MSI on all AKS agent pools
-      node_resource_group=$(az aks show -g ${AKS_CLUSTER_NAME} -n ${RESOURCE_GROUP} --query "nodeResourceGroup")
-      for vm in ${az vmss list -g $nodeResourceGroup --query "[].name" -o tsv}
-      do
-        az vmss identity assign -g $node_resource_group -n $vm --identities ${IDENTITY_RESOURCE_ID}
-      done
-      ```
-
-4. Deploy ratify helm chart
+
+8. Deploy from local helm chart:
 ```shell
-export TENANT_ID={{ Tenant Id }}
-helm install ratify \
-    ratify/ratify --atomic \
-    --set oras.authProviders.azureManagedIdentityEnabled=true \
-    --set azureManagedIdentity.clientId=${IDENTITY_CLIENT_ID} \
-    --set azureManagedIdentity.tenantId=${TENANT_ID}
+# Setup Gatekeeper first.
+helm repo add gatekeeper https://open-policy-agent.github.io/gatekeeper/charts
+
+helm install gatekeeper/gatekeeper  \
+  --name-template=gatekeeper \
+  --namespace gatekeeper-system --create-namespace \
+  --set enableExternalData=true \
+  --set validatingWebhookTimeoutSeconds=5 \
+  --set mutatingWebhookTimeoutSeconds=2
+
+# Deploy Ratify
+helm install ratify ./charts/ratify --atomic \
+  --namespace ${RATIFY_NAMESPACE} \
+  --set-file provider.tls.crt=${CERT_DIR}/server.crt \
+  --set-file provider.tls.key=${CERT_DIR}/server.key \
+  --set provider.tls.cabundle="$(cat ${CERT_DIR}/ca.crt | base64 | tr -d '\n')" \
+  --set oras.authProviders.azureWorkloadIdentityEnabled=true \
+  --set azureWorkloadIdentity.clientId=${IDENTITY_CLIENT_ID} \
+  --set akvCertConfig.enabled=true \
```

Comment:
Considering this doc title is Auth Provider, I would suggest removing the Gatekeeper and Ratify deployment steps from this section and adding related links instead.